### DISCUSSION TITLE
Project workflows Bug

### DISCUSSION THREAD

**Discussion by tpatartmajeur** [community/community#32295] [Projects and Issues] [answered]
Hello,
I am having a problem configuring the workflow for my project.

I can't configure "Item closed".

This error occurs when I choose the feature and backlog templates at project creation.
![Screenshot_20220908_135033](https://user-images.githubusercontent.com/51753505/189115177-af862c55-2e12-423c-b5f0-5667e8cecbe8.png)

![screenshot-github com-2022 09 08-13_50_03](https://user-images.githubusercontent.com/51753505/189115028-61d1f885-387d-402b-9115-4c6762f1708c.png)

**Comment by RakeshSangem** [comment, marked as answer]
I am also facing the same issue. It should be fixed ASAP! It's quite frustrating task to move cards.

**Comment by JandenMa** [comment]
Same issue here. It should be fixed ASAP pls!!

**Reply by RakeshSangem** [reply]
Yeah, someone should work on it ASAP!
